.github/workflows/PR-build.yaml

```diff
@@ -0,0 +1,31 @@
+name: PR build
+
+on:
+    workflow_dispatch:
+    pull_request:
+        branches: [main]
+
+jobs:
+    Build-wrapping-artifacts:
+        name: Build wrapping arifacts
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkin Code
+              uses: actions/checkout@v4
+            
+            - name: Set up JDK 11
+              uses: actions/setup-java@v4
+              with:
+                java-version: '11'
+                distribution: 'temurin'
+                cache: maven
+            
+            - name: Building artefacts
+              run: |
+               mvn clean install -Dgpg.skip=true \

```

Comment:
Nice, yes as you've provided the galasa.release.repo but for PRs we don't want to actually deploy there, `clean install` works better here